src/platform/keyboard_input.c

```diff
@@ -19,6 +19,16 @@ static int is_alt_down(SDL_KeyboardEvent *event)
     return (event->keysym.mod & KMOD_ALT) != 0;
 }
 
+static int is_key_currently_down(SDL_Scancode key)
+{
+    int numkeys = 0;
+    Uint8* key_state = SDL_GetKeyboardState(&numkeys);
+    if (key > numkeys - 1)
```

Comment:
Use curly braces please for if/else statements, also for the other changes in this file.